spec/factories/prolong_token_wizard.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :prolong_token_wizard do
+    after(:build) do |prolong_token_wizard, _|
+      prolong_token_wizard.token = create(:token) if prolong_token_wizard.token.nil?
+    end
```

Comment:
```suggestion
    association :token, strategy: :create 
```

t'as forcément besoin que sur un build du wizard t'es un create de token dans les tests ? sinon:

```ruby
  token
```